conf/test/manual.csv

```diff
@@ -86,3 +86,7 @@ TestStep,,,,3,"apply update task using ""swupd_update -v -c"" and make sure upda
 TestStep,,,,4,"do ""swupd verify"" to check system is correct",check system should be in good status,,,,,,,
 TestScript,multilib_integration,SCM / Build Tools,check if the multilib can be built by ostro build environment,1,get the ostro-os git repo,,FVT,ready,IOTOS-716,,,,
 TestStep,,,,2,"add ""require conf/ostro-multilib.conf"" to your build/conf/local.conf, then build ostro-image",the ostro-image can be built successfully,,,,,,,
+TestScript,secure_boot_support,Security,check if Ostro supports secure boot,1,Create and sign certificates; install signing key to bios following the docs fom ostroproject/meta-ostro/blob/master/doc/howtos/uefi-secure-boot.rst,,FVT,ready,IOTOS-409,,,,
+TestStep,,,,2,Boot an unsigned image (default), image fails to boot: "Invalid signature detected. Check Secure Boot Policy in Setup",,,,,,,
```

Comment:
The quoting is not correct for github web parser:
for example, for the step 2, please change it to
"Boot an unsigned image (default), image fails to boot: ""Invalid signature detected. Check Secure Boot Policy in Setup"""
otherwise, the github parser will report error and cannot list CSV as table correctly.